From: SVN-Git Migration <python-modules-team@lists.alioth.debian.org>
Date: Mon, 12 Oct 2015 15:17:20 -0700
Subject: use_local_mathjax

Setting mathjax_path variable to use the local mathjax javascript library
Patch-Name: use_local_mathjax.patch
---
 doc/source/conf.py | 1 +
 1 file changed, 1 insertion(+)

Index: scipy/doc/source/conf.py
===================================================================
--- scipy.orig/doc/source/conf.py	2024-11-21 15:05:00.527871824 +0100
+++ scipy/doc/source/conf.py	2024-11-21 15:05:00.515871719 +0100
@@ -273,6 +273,7 @@
 htmlhelp_basename = 'scipy'
 
 mathjax_path = "scipy-mathjax/MathJax.js?config=scipy-mathjax"
+mathjax_path='/usr/share/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML'
 
 # -----------------------------------------------------------------------------
 # Intersphinx configuration
